src/ec/packages/ec-component-search-form/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@ecl-twig/ec-component-search-form",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.0.1-alpha",
+  "description": "ECL Twig - EC Search Form",
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@ecl/ec-specs-search-form": "~2.1.0"
```

Comment:
please add a devDependency to vanilla EC component too

Reply:
Added in https://github.com/ec-europa/ecl-twig/pull/28/commits/72483ba8d0c2d29290bdd9a64af43fa2ae0d59e7